source/light_utils.h

```diff
@@ -8,15 +8,23 @@
     Structs
 */
 // Struct defining an LED and its RGB color components
-typedef struct {
-  uint8_t red, green, blue;
+typedef union {
+  struct {
+    /* Little endian ordering to match uint32_t */
+    uint8_t blue, green, red;
+    uint8_t align;
+  } p;
```

Comment:
why `p`?

Reply:
"parts" I wanted it to be short. pv - parts vector. But I'm not particularly attached to it.

Reply:
People on discord are awesome. Got one easy fix already and I wonder about a better dimming. But that's a bit bigger change and maybe separately would be better.